core/server/pom.xml

```diff
@@ -35,14 +35,16 @@
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
     </dependency>
+    <!-- alluxio-core-common should come before alluxio-core-client as alluxio-core-client shade some dependencies
```

Comment:
please also add a TODO to remove the alluxio-core-client dependency as having core-common be in the dependency tree twice -- once shaded through alluxio-core-client and once unshaded directly -- is error prone